<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditUserWidget</class>
 <widget class="QWidget" name="EditUserWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>464</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string notr="true">EditUserWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>User Information</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Full Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="lineFullname">
        <property name="text">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="linePassword">
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Password (Again):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="linePasswordAgain">
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="2">
       <widget class="QLineEdit" name="lineUsername">
        <property name="text">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" rowspan="2">
    <widget class="QGroupBox" name="advancedGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Home Directory:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="lineHomeDir">
        <property name="text">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Shell:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QComboBox" name="comboShell">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QCheckBox" name="checkAdmin">
        <property name="text">
         <string>Give administrator rights</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="locale">
         <locale language="Turkish" country="Turkey"/>
        </property>
        <property name="text">
         <string>User Id:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinId">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>-1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="checkAutoId">
        <property name="text">
         <string>Auto</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushAdvanced">
     <property name="text">
      <string>Advanced</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelSign">
       <property name="minimumSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelWarning">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" rowspan="9">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabAuthorizations">
      <attribute name="title">
       <string>Authorizations</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="0">
        <widget class="QTreeWidget" name="treeAuthorizations">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
         <property name="columnCount">
          <number>1</number>
         </property>
         <column>
          <property name="text">
           <string notr="true">Policy</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="pushAuth">
         <property name="text">
          <string>Apply this policy to all actions</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QWidget" name="authGroup" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QRadioButton" name="radioAuthDefault">
            <property name="text">
             <string>Use system settings</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioAuthYes">
            <property name="text">
             <string>Authorize user</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioAuthNo">
            <property name="text">
             <string>Deny user</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="KTreeWidgetSearchLine" name="filterAuthorizations">
         <property name="clickMessage">
          <string>Start typing to filter</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabGroups">
      <attribute name="title">
       <string>Groups</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0" colspan="2">
        <widget class="QListWidget" name="listGroups">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Main Group:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboMainGroup">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="KListWidgetSearchLine" name="filterGroups">
         <property name="clickMessage">
          <string>Start typing to filter</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KTreeWidgetSearchLine</class>
   <extends>KLineEdit</extends>
   <header>ktreewidgetsearchline.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KListWidgetSearchLine</class>
   <extends>KLineEdit</extends>
   <header>klistwidgetsearchline.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineFullname</tabstop>
  <tabstop>linePassword</tabstop>
  <tabstop>linePasswordAgain</tabstop>
  <tabstop>lineUsername</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushAdvanced</sender>
   <signal>clicked(bool)</signal>
   <receiver>advancedGroup</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>46</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
